services/libs/data-access-layer/src/conversations/ilp.ts

```diff
@@ -17,6 +17,8 @@ export async function insertConversations(
   const now = Date.now()
 
   if (conversations.length > 0) {
+    ilp = getClientILP()
```

Comment:
Everything ok except I would cache the already created instance and not create instance every time the function is called.

Reply:
Good point :+1:  I changed it to check if the variable has been initialised first and only do so if it hasn't yet.